fe-games/src/components/FullReview.jsx

```diff
@@ -1,7 +1,33 @@
+import { useState } from "react";
+import { useParams, Link } from "react-router-dom";
+
 const FullReview = () => {
+    const [review, setReview] = useState({});
+    const [comments, setComments] = useState([]);
+    const { review_id } = useParams();
+    const date = new Date(review.created_at);
+
+    fetch(`https://michaels-games.onrender.com/api/reviews/${review_id}`)
+    .then(response => response.json())
+    .then(data => {
+        setReview(data.review);
+    })
+
+    fetch(`https://michaels-games.onrender.com/api/reviews/${review_id}/comments`)
+    .then(response => response.json())
+    .then(data => {
+        setComments(data.comments);
+    })
+
     return (
-        <div>
-            <p>FullReview</p>
+        <div className="FullReview">
+            <Link to="/" ><button className="LeftButton">🏠</button></Link>
+            <h2>{review.title}</h2>
+            <img className="ReviewImage" src={review.review_img_url} alt="image related to review" />
```

Comment:
In alt text you don't need to write the word "image". Screen readers will read the img tag out loud so it'll get repeated if you write it in yourself.

Reply:
Okay cool ty